NICE.Identity.Management/ClientApp/src/react-app-env.d.ts

```diff
@@ -0,0 +1,4 @@
+/// <reference types="react-scripts" />
+
+declare module "@nice-digital/nds-forms";
```

Comment:
You can remove these 2 module declarations as we'll be providing type defs as part of each package